### DISCUSSION TITLE
Who use APIMAN as API management ?

### DISCUSSION THREAD

**Discussion by mekhaba** [apiman/apiman#1438] [General]
Hi to all, 

Actually, I'm doing a benchmark for open source API management and I want to use one of themes in my company.
I've experimented with Java and Spring.

I'm searching for clients who use APIMAN as API management to be sure of my choice. 

Can anyone, please, give some shortlist for APIMAN clients ( references I mean )

Thanks in advance.

**Comment by msavy** [comment]
Apiman has a large number of users, but most do not publicly share that they use Apiman. 

Perhaps we can try to encourage more people to talk about it, but it's difficult in an open source project.
